examples/hello_gpu.py

```diff
@@ -0,0 +1,33 @@
+"""numba/cuda-on-ray equivalent of hello world"""
+import math
+
+import numpy as np
+import ray
+from numba import cuda
+
+
+def test(v: np.array):
+    x, y = cuda.grid(2)
+    v[x, y] = 1
+
+
+@ray.remote(num_gpus=1)
+def cuda_test(width, height):
```

Comment:
Rename this function or add PyDoc please

Reply:
actually, on second thought, this one seems quite accurate...